dev/MRTCore/mrt/mrm/include/mrm/platform/CoreQualifierTypes.h

```diff
@@ -60,7 +60,7 @@ class DXFeatureLevelQualifierType : public EnumerationQualifierType
 
     virtual ~DXFeatureLevelQualifierType() {}
 
-    HRESULT Evaluate(_In_ const IQualifier* pQualifier, _In_ PCWSTR pValue, _Outptr_ double* score) const;
+    HRESULT Evaluate(_In_ const IQualifier* pQualifier, _In_ PCWSTR pValue, _Out_ double* score) const;
```

Comment:
this fix was not clear. More detail please.

Reply:
Sure, for context, here's the original error:
```
Tool: PREfast: Rule: C6504 ((Unknown Rule Name)). 
Invalid annotation: 'Null' property may only be used on values of pointer, pointer-to-member, array, or reference type: Function 'Evaluate' _Param_(3). Actual type 'double'.
```
Fixed by matching with another 2 instances of the Evaluate() method in "class ContrastQualifierType" (line 16) and "class ScaleQualifierType" (line 44) that did _not_ trigger this PREfast issue.

I think the right use of _Outptr_ in this context should look like "_Outptr_ double** score". PREfast spotted the missing "*", but the code author likely meant to use _Out_ like in the other 2 instances in the same source, instead of really meaning to say "_Outptr_ double** score".